### PR TITLE
Dirty hack to temporarily fix UCR pillow table rebuilding bug

### PR DESCRIPTION
**Pull request by NoahCarnahan**
A demo to important stakeholders tomorrow requires that the reports be functional. Due to this UCR pillow bug, the data sources keep getting emptied: https://manage.dimagi.com/default.asp?252832

This needs to be deployed today
@emord 
buddy @gcapalbo

### CODE REVIEW

File: corehq/apps/userreports/pillow.py [hunk 1]
```diff
@@ -194,14 +194,16 @@ def _rebuild_es_tables(self, adapters):
...
-        config = adapter.config
-        if not config.is_static:
-            latest_rev = config.get_db().get_rev(config._id)
-            if config._rev != latest_rev:
-                raise StaleRebuildError('Tried to rebuild a stale table ({})! Ignoring...'.format(config))
-        adapter.rebuild_table()
-        if self.auto_repopulate_tables:
-            rebuild_indicators.delay(adapter.config.get_id)
+        # https://manage.dimagi.com/default.asp?252832
+        if adapter.config.get_id not in ("static-enikshay-episode", "static-np-migration-3-episode"):
```
Comment (emord) [inline corehq/apps/userreports/pillow.py]: nitpick I think it'd be better to use `if adapter.config.get_id in (...): return` to make it more obvious

**Review by emord — APPROVED** [review]
looks fine for nwo

File: corehq/apps/userreports/pillow.py (hunk 1 above)
Comment (NoahCarnahan) [inline corehq/apps/userreports/pillow.py]: updated

**Review by emord — APPROVED** [review]

**Comment by NoahCarnahan** [comment]
This didn't work

**Pull request merged**